.github/workflows/test-action-windows.yml

```diff
@@ -88,7 +86,7 @@ jobs:
     steps:
       - uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
+          python-version: 3.10.14
```

Comment:
We created a new variable "gh-actions-python-version" above... would we want to use that here?